guides/common/modules/proc_upgrading-project-in-place-using-leapp.adoc

```diff
@@ -50,6 +61,39 @@ On {RHEL}, enable the `rhel-7-server-extras-rpms` repository:
 ----
 # {package-install-project} leapp leapp-repository
 ----
+ifdef::satellite[]
+. For Leapp to perform the upgrade in a disconnected environment, download the metadata and manually extract, as described in https://access.redhat.com/articles/3664871[Leapp utility metadata in-place upgrades of RHEL for disconnected upgrades].
+
+. Set up the following repositories to perform the upgrade in a disconnected environment:
+.. `/etc/yum.repos.d/rhel8.repo`:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+[BaseOS]
+name={RepoRHEL8BaseOS}
+baseurl=http://_server.example.com_/rhel8/BaseOS/
+[AppStream]
+name={RepoRHEL8AppStream}
+baseurl=http://_server.example.com_/rhel8/AppStream/
+----
++
+.. `/etc/yum.repos.d/{project-context}.repo:`
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+[{RepoRHEL8ServerSatelliteServerProductVersion}]
+name={RepoRHEL8ServerSatelliteServerProductVersion}
+baseurl=http://_server.example.com_/sat6/Satellite/
+[{RepoRHEL8ServerSatelliteMaintenanceProductVersion}]
+name={RepoRHEL8ServerSatelliteMaintenanceProductVersion}
+baseurl=http://_server.example.com_/sat6/Maintenance/
```

Comment:
```suggestion
baseurl=http://_{foreman-example-com}_/rhel8/BaseOS/
[AppStream]
name={RepoRHEL8AppStream}
baseurl=http://_{foreman-example-com}_/rhel8/AppStream/
----
+
.. `/etc/yum.repos.d/{project-context}.repo:`
+
[options="nowrap", subs="+quotes,verbatim,attributes"]
----
[{RepoRHEL8ServerSatelliteServerProductVersion}]
name={RepoRHEL8ServerSatelliteServerProductVersion}
baseurl=http://_{foreman-example-com}_/sat6/Satellite/
[{RepoRHEL8ServerSatelliteMaintenanceProductVersion}]
name={RepoRHEL8ServerSatelliteMaintenanceProductVersion}
baseurl=http://_{foreman-example-com}_/sat6/Maintenance/
```

Reply:
I'm not sure about the example server URL here.
The following NOTE says that the ISO must be mounted on a different machine than the "foreman server" (?).
Perhaps the example server should reflect this?
Something like: `server-with-mounted-iso.example.com/sat6/...` ?

Reply:
I see, thanks for the correction. Please ignore this suggestion, then.